components/common/homepage/extra/exploreAfrilearn.jsx

```diff
@@ -1,73 +1,75 @@
 import {
   // MouseEvent,
   useState,
-} from "react";
-import styles from "../homepage.module.css";
-import VideoBox from "./exploreVideoBox";
-import StatsBox from "./statisticsBox";
+} from 'react'
+import styles from '../homepage.module.css'
+import VideoBox from './exploreVideoBox'
+import StatsBox from './statisticsBox'
 
-const ExploreAfrilearn = () => {
-  const [selectedOption, setSelectedOption] = useState(1);
+const ExploreAfrilearn = ({ userRole }) => {
+  const [selectedOption, setSelectedOption] = useState(1)
```

Comment:
Please note that this image should be changed as agreed.
![image](https://user-images.githubusercontent.com/10332499/203933038-b0c5a267-5f8b-4c1d-abfe-a1fe724e709f.png)